### DISCUSSION TITLE
Github contributions not matching my commits

### DISCUSSION THREAD

**Discussion by David-orovs** [community/community#45117] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi everyone, i seem to be having a problem, when i make a commit, my Github contributions dont update, how can i solve this

**Comment by radar-blip** [comment]
If your commits are in a private repository, they wont count unless you make changes in your settings.

**Here (on profile)**:
![conts](https://user-images.githubusercontent.com/39196767/213982566-a81e7856-e9ca-4afd-8ce4-dffb453321bf.png)

**and here (in settings)**:
![conts2](https://user-images.githubusercontent.com/39196767/213982605-0fff4ee3-df11-45f9-98e2-7884f8c47c58.png)

**Reply by David-orovs** [reply]
it didnt solve my issue, the issue is my commits do not reflect

**Reply by radar-blip** [reply]
Okay, maybe this will help: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile#contributions-that-are-counted

**Comment by perezz281** [comment]
I reply

On Mon, Jan 23, 2023 at 2:14 AM David ***@***.***> wrote:

> Okay, maybe this will help:
> https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile#contributions-that-are-counted
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/45117#discussioncomment-4753683>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A5NCDLFMN3YH7WW5NMULVU3WTYVTTANCNFSM6AAAAAAUDPPRGM>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***
> com>
>
-- 
Anthony Perez

**Comment by David-orovs** [comment]
thanks @perezz281 @linuxlawson

**Reply by radar-blip** [reply]
You're welcome. Hope it helped!
